packages/turf-transform-scale/index.js

```diff
@@ -59,18 +61,24 @@ module.exports = function (geojson, factor, origin, mutate) {
  * @param {Feature|Geometry} geojson GeoJSON Feature/Geometry
  * @param {number} factor of scaling, positive or negative values greater than 0
  * @param {string|Geometry|Feature<Point>|Array<number>} [origin="centroid"] Point from which the scaling will occur (string options: sw/se/nw/ne/center/centroid)
+ * @param {boolean} [mutate=false] allows GeoJSON input to be mutated
```

Comment:
Here `mutate` does not actually define if the feature is going to be mutated or not, that was handled before.
I can't find a good description though 😅